CHANGES.rst

```diff
@@ -275,6 +275,8 @@ API Changes
 
   - The `astropy.vo.samp` package no longer supports SSL. [#6201]
 
+  - The `astropy.vo.samp` package has been moved to `astropy.samp`. [#6213]
```

Comment:
Nitpick: This entry and the one for #6201 above are a bit confusing. Maybe merge them into a single entry and reword in a more coherent manner?